ext/tests/return_expanded_params_internal.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+Check if pre hook can expand and then return $params of internal function
+--EXTENSIONS--
+opentelemetry
+--FILE--
+<?php
+\OpenTelemetry\Instrumentation\hook(null, 'array_slice', function($obj, array $params) {
+    $params[2] = 1; //only slice 1 value, instead of "remainder"
+    return $params;
+},
+fn() => null);
```

Comment:
Is this needed?

Reply:
I don't think so. I will remove.

Reply:
Actually, it _is_ important. It was part of the original bug report, and it doesn't fail without a post callback being registered.